litr-demo/src/main/java/com/linkedin/android/litr/demo/data/TransformationPresenter.java

```diff
@@ -556,33 +560,46 @@ public void transcodeAudio(@NonNull SourceMedia sourceMedia,
                 TimeUnit.MILLISECONDS.toMicros((long) (trimConfig.range.get(1) * 1000)))
                 : new MediaRange(0, Long.MAX_VALUE);
 
-        TransformationOptions transformationOptions = new TransformationOptions.Builder()
-                .setGranularity(MediaTransformer.GRANULARITY_DEFAULT)
-                .setSourceMediaRange(mediaRange)
-                .build();
+        try {
+            String targetMimeType = targetMedia.writeToWav ? "audio/raw" : "audio/mp4a-latm";
+            MediaTarget mediaTarget = targetMedia.writeToWav
+                    ? new WavMediaTarget(targetMedia.targetFile.getPath())
```

Comment:
minor: Should we change the extension to wav in these cases?

Reply:
I tried doing that, but for some reason transcoded file becomes inaccessible.